ably/realtime/connection.py

```diff
@@ -219,6 +233,11 @@ async def ping(self):
                                               "id": self.__ping_id})
         else:
             raise AblyException("Cannot send ping request. Calling ping in invalid state", 40000, 400)
+        try:
+            await asyncio.wait_for(self.__ping_future, self.options.realtime_request_timeout)
+        except asyncio.TimeoutError:
+            raise AblyException("Realtime request timeout", 504, 50003)
```

Comment:
Since this is user-facing it would be better to be more descriptive here
```suggestion
            raise AblyException("Timeout waiting for ping response", 504, 50003)
```